ferretdb/ferretdb.go

```diff
@@ -100,11 +102,22 @@ func New(config *Config) (*FerretDB, error) {
 	}
 
 	lOpts := &logging.NewHandlerOpts{
-		Base:       "console",
-		Level:      logLevel,
 		SkipChecks: true,
+		Level:      logLevel,
+	}
```

Comment:
We often avoid unnecessary `else` statement. Also I don't think we need to instantiate the logger separately for example. Could it be simplified something like below?
```
	var embeddedHandler slog.Handler
	logBase := "console"

	if config.Logger != nil {
		logBase = "embeddable"
		embeddedHandler = config.Logger.Handler()
		logOutput = nil
	}

	lOpts := &logging.NewHandlerOpts{
		Base:       logBase,
		Level:      logLevel,
		Handler:    embeddedHandler,
		SkipChecks: true,
	}

	logger := logging.WithName(logging.Logger(logOutput, lOpts, ""), "ferretdb")

```